test_addonfactory_splunk_conf_parser_lib.py

```diff
@@ -175,40 +183,43 @@ def test_options(self):
         expected_options = ["__name__", "EVAL-field1", "FIELDALIAS-field2"]
         self.assertEqual(expected_options, parser.options("source"))
 
-    def test_item_dict(self):
-        conf = """
-#
-# Very big copyright comment.
-#
-[source]
-# This is a very simple field aliasing.
-FIELDALIAS-field1 = field2 AS field1
-EVAL-field3 = case(isnotnull(field4), "success",\
-                   isnotnull(field5), "failure")
-
-[sourcetype:deprecated]
-EVAL-deprecated_field = "old_value"
+    def test_item_dict_when_preserve_comment_False(self):
+        parser = conf_parser.TABConfigParser()
+        parser.read(CONF_PATH)
```

Comment:
nit: you could inline the `conf` and parametrize the test to stay consistent with other tests but totally up to you